src/main/electron.ts

```diff
@@ -14,10 +14,11 @@ import IPCMessages from './ipc/messages'
 import { setMenu } from './menu'
 
 export const IS_DEV = isDev && process.env.NODE_ENV !== 'production'
+export const PORT = process.env.PORT || 3000
```

Comment:
^ neeeded to run ASGARDEX side by side with other web application (e.g. previous `ASGARD` wallet)